lib/i18n/tasks/translators/openai_translator.rb

```diff
@@ -52,7 +52,12 @@ def no_results_error_message
     private
 
     def translator
-      @translator ||= OpenAI::Client.new(access_token: api_key)
+      @translator ||= OpenAI::Client.new(
+        access_token: api_key,
+        # Highly recommended in development, so you can see what errors OpenAI is returning.
+        # Not recommended in production because it could leak private data to your logs.
+        log_errors: ENV.fetch('RAILS_ENV', ENV.fetch('RACK_ENV', '')) == 'development'
```

Comment:
`RAILS_ENV` and `RACK_ENV` are usually not defined when running i18n-tasks.
While it supports Rails, it does it entirely via static analysis, and does not load rails itself.

It is possible of course to load i18n-tasks as a library from within rails, in which case it'd be good to respect this.

However, most of the time, this string will be empty. So let's also log errors when it's empty.

Reply:
Ok i see. Anyways, in your README.md most of the time the gem will be installed only in the group :development anyways.